contentcuration/contentcuration/utils/celery/tasks.py

```diff
@@ -178,7 +177,10 @@ def find_incomplete_ids(self, signature):
         :return: A TaskResult queryset
         :rtype: django.db.models.query.QuerySet
         """
-        return self.find_ids(signature).filter(status__in=states.UNREADY_STATES)
+        # Get the filtered task_ids from CustomTaskResult model
+        filtered_task_ids = self.find_ids(signature)
+        # Use the filtered task_ids to query TaskResult model
+        return self.TaskModel.objects.filter(task_id__in=filtered_task_ids, status__in=states.UNREADY_STATES)
```

Comment:
Small regression here, this should return a QuerySet with `.values_list("task_id", flat=True)`, which you fixed in the implementation below, but this method is intended to return IDs